strings_lab.js

```diff
@@ -0,0 +1,66 @@
+// let str = "johanne";
+// console.log(str[0])
+
+// console.log("---------------")
+//
+// let str = "johanne";
+// console.log(str.length)
+//
+// console.log("-----------------")
+
+// let str = "johanne";
+// console.log(str[str.length - 1])
+
+// console.log("----------------------")
+
+
+// let str = 'johanne';
+// str = str.slice(0, 6) + str[str.length - 1].toUpperCase()
+// console.log(str)
+//
+// let str = 'johanne';
+// str = str.slice(0,str.length-1 ) + str[str.length - 1].toUpperCase()
+// console.log(str)
+
+
+
+
+// console.log("-------------------------")
+//
+// // let variableAmount = 10
+// // if (variableAmount = 1000000) {
+// //   console.log(1000000 + ' dollars' + ' pinky')
+// // }else {
+// //   console.log(10 + ' dollars' )
+// // }
+//
+// console.log("-------------------------")
+//
+// let str = "read"
+// if (str.length < 3) {
+//   console.log("unchanged")
+// } else if (str.length >= 3) {
+//   console.log(str + "ing");
+// } else {
+//   console.log(str + "ly")
+// }
+
+// console.log("--------------------")
+
+// let withoutLast = "It is very hot"
```

Comment:
If I change the variable withoutLast to "It wasn't very hot". I would not give me "It wasn't very" as output. Perhaps we can try to solve this with split method. For example: 

let str = 'The quick brown fox jumps over the lazy dog.'
let words = str.split(' ');
words.pop()
console.log(words);